### PR TITLE
MedianDataFeedOracle: setResult() must contain each approved dataFeed…

### PR DESCRIPTION
**Pull request by emaG3m**
… without duplicates

### CODE REVIEW

File: contracts/DataFeedOracles/MedianDataFeedOracle.sol [hunk 1]
```diff
@@ -31,10 +33,16 @@ contract MedianDataFeedOracle is Initializable, DataFeedOracleBase {
...
     for (uint i = 0; i < _dataFeeds.length; i++) {
        require(approvedDataFeeds[address(_dataFeeds[i].dataSource)], "Unauthorized data feed.");
        require(_dataFeeds[i].latestResultDate() > latestResultDate(), "Stale data.");
 
+       for (uint j = i + 1; j < _dataFeeds.length; j++) {
+         require(_dataFeeds[i] != _dataFeeds[j], "Duplicate data feeds prohibited");
+       }
+
        if(i != _dataFeeds.length - 1) {
          require(uint256(_dataFeeds[i].latestResult()) <= uint256(_dataFeeds[i+1].latestResult()), "The dataFeeds is not sorted.");
```
Comment (mikec) [inline contracts/DataFeedOracles/MedianDataFeedOracle.sol]: might wanna update this require message

File: test/DataFeedOracles/MedianDataFeedOracle.test.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ const ORACLE_RESULT = PRICES.map(
       z => [ DATES[y.indexOf(z)], numberToBytes32(z * Math.pow(10, 18) )]
 ))
 
-contract('initialize MedianDataFeedOracle', (accounts) => {
-  let oracles
+contract.only('MedianDataFeedOracle', (accounts) => {
```
Comment (mikec) [inline test/DataFeedOracles/MedianDataFeedOracle.test.js]: forgot to remove the `.only`!

Comment (emaG3m) [reply test/DataFeedOracles/MedianDataFeedOracle.test.js]: :o

**Comment by mikec** [comment]
@emaG3m thanks for the changes, 👍 👍

**Pull request merged**